client/src/extension.ts

```diff
@@ -53,7 +58,7 @@ namespace Configuration {
             }
             result.push({
                 autocheck: config.get<boolean>('autocheck'),
```

Comment:
better way to represent `path.join(__dirname, '..', '..'` ? seems a little ugly

Reply:
#18